script/deploy/mainnet/011_OgnOgvMigrationScript.sol

```diff
@@ -54,102 +64,94 @@ contract OgnOgvMigrationScript is BaseMainnetScript {
         console.log("- Migrator init");
         migratorProxy.initialize(address(migratorImpl), Addresses.TIMELOCK, "");
 
-        _buildGnosisTx();
+        _buildGovernanceProposal();
     }
 
-    function _buildGnosisTx() internal {
+    function _buildGovernanceProposal() internal {
         Timelock timelock = Timelock(payable(Addresses.TIMELOCK));
 
         address ognRewardsSourceProxy = deployedContracts["OGN_REWARDS_SOURCE"];
         address veOgvImpl = deployedContracts["VEOGV_IMPL"];
 
-        govFive.setName("OGV Migration to OGN");
-        // Todo: Fuller description
-        govFive.setDescription("Deploy OGV-OGN migration contracts and revoke OGV Governance roles");
+        govProposal.setDescription("Deploy OGV-OGN migration contracts and revoke OGV Governance roles");
```

Comment:
Since this will be one of the most impactful governance proposals in while, it could be good to go into more details of what this proposal will do.